packages/css/src/types.ts

```diff
@@ -120,7 +120,15 @@ export interface Composition {
   classList: string;
 }
 
-type IdentOption = 'short' | 'debug';
+type CustomIdentFunction = (params: {
+  hash: string;
+  debugId?: string;
+  filePath?: string;
```

Comment:
`filePath` will always be available
```suggestion
  filePath: string;
```

Reply:
Done